config/sync/field.field.media.stanford_gallery_images.su_gallery_image.yml

```diff
@@ -0,0 +1,38 @@
+uuid: 079c2adc-db3f-40b0-a919-fa97da03717b
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.media.su_gallery_image
+    - media.type.stanford_gallery_images
+  module:
+    - image
+id: media.stanford_gallery_images.su_gallery_image
+field_name: su_gallery_image
+entity_type: media
+bundle: stanford_gallery_images
+label: 'Gallery Image'
+description: ''
+required: false
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings:
+  file_directory: media/image/gallery
+  file_extensions: 'png gif jpg jpeg'
+  max_filesize: 5MB
```

Comment:
I bet we get complaints about this limit being too low.  I think it's a reasonable size, but users always want to upload giant images.

Reply:
occasionally, but since the image styles being generated are less than 1MB, their argument is moot. :smile: